### PR TITLE
Activate LSP for schema and schema extension files

### PR DESCRIPTION
**Pull request by tobias-tengler**
This is in preparation for adding LSP support for schema and schema extension files.

### CODE REVIEW

**Comment by captbaritone** [comment]
Awesome! I'm curious how/if you were able to verify these changes. This LSP stuff can be hard to test (especially when they require changes to the VSCode extension as well)

**Comment by tobias-tengler** [comment]
Yes, I was able to verify it. I started the extension development host and launched my lsp playground repository, which is configured to point to the locally built relay binary. I then observed the extension displayed the `Relay: ready` message, when I opened the configured `schema.graphql` file.

File: compiler/crates/relay-lsp/src/server/lsp_state.rs [hunk 1]
```diff
@@ -521,12 +530,34 @@ impl<TPerfLogger: PerfLogger + 'static, TSchemaDocumentation: SchemaDocumentatio
             js_server.process_js_source(uri, text);
         }
```
Comment (captbaritone) [inline compiler/crates/relay-lsp/src/server/lsp_state.rs:531]: Should this block live in the FileGroup::Source match case, since it's js-specific?

**Review by captbaritone — CHANGES_REQUESTED** [review]
Looks good! Just one qustion

**Comment by tobias-tengler** [comment]
Addressed :)

**Pull request closed without merging**